packages/wrangler/src/pages/upload.tsx

```diff
@@ -353,11 +353,13 @@ export const upload = async (
 };
 
 // Decode and check that the current JWT has not expired
-function isJwtExpired(token: string): boolean | undefined {
+export const isJwtExpired = (token: string): boolean | undefined => {
```

Comment:
ideally this gets pulled out into some common place, but nbd.

Reply:
+1